crates/neon/src/sys/string.rs

```diff
@@ -30,6 +30,24 @@ pub unsafe fn data(env: Env, out: *mut u8, len: isize, value: Local) -> isize {
     read.assume_init() as isize
 }
 
+pub unsafe fn utf16_len(env: Env, value: Local) -> isize {
```

Comment:
It looks like a mistake that we are casting to `isize` in the UTF-8 functions. I would prefer to fix them and have the breaking change prior to 1.0.

Reply:
I saw that and wondered why `isize` over `usize` here, but simply followed the existing code to keep it consistent. I'll change both to `usize` then.

Reply:
It was likely a carry over from the original C++/NAN backend and matched the types defined there. We were trying to allow swapping the backend without changing the Neon API. However, that backend has been removed leading up to 1.0 and it would be preferred to fix this.

Thanks for taking that on! If you prefer to hold off, we can file a follow-up.